README.md

```diff
@@ -1 +1,40 @@
-# proofreader-python
\ No newline at end of file
+# Proofreader Python
+proofreader-python is a tool for enforcing opinionated coding standards and conventions through static analysis of the code.
+
+It uses:
+- [Pylint](https://github.com/PyCQA/pylint) 
+- [Flake8](https://github.com/PyCQA/flake8)
+    
+
+Dependencies
+------------
+
+* Python 2.7
+* Python >=3.5
+
+Installation
+------------
+
+`pip install proofreader` ** not yet implemented **
```

Comment:
what is needed for that? Only the upload to PyPI?

Reply:
Yes, just the upload to PyPi.